src/Bicep.LangServer/Handlers/BicepDefinitionHandler.cs

```diff
@@ -157,7 +157,7 @@ private Uri GetDocumentLinkUri(ISourceFile sourceFile, ModuleReference moduleRef
             var fullyQualifiedReference = WebUtility.UrlEncode(moduleReference.FullyQualifiedReference);
 
             // Encode the source file path as a path and the fully qualified reference as a fragment.
-            return new Uri($"bicep-cache:///{sourceFilePath}#{fullyQualifiedReference}");
```

Comment:
On macOS and Linux, `sourceFilePath` starts with `/`. As a result, the `Path` in the URI begin with `//`, which violates https://www.rfc-editor.org/rfc/rfc3986#page-16:
> When authority is not present, the path cannot begin with two slash characters ("//")

and causes `System.UriFormatException`.

Remove the empty Authority section from the URI fixes the issue.